.github/workflows/build.yaml

```diff
@@ -19,10 +19,10 @@ jobs:
           - cfg: {name: 'gcc-arm',           runson: 'ubuntu-22.04-arm', container: 'ubuntu:noble'}
           - cfg: {name: 'clang-arm',         runson: 'ubuntu-22.04-arm', container: 'ubuntu:noble',
                   cmake: '-DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++'}
-          - cfg: {name: 'gcc10',             runson: 'ubuntu-latest',    container: 'debian:bullseye'}
           - cfg: {name: 'gcc11',             runson: 'ubuntu-latest',    container: 'ubuntu:jammy'}
           - cfg: {name: 'gcc12',             runson: 'ubuntu-latest',    container: 'debian:bookworm'}
           - cfg: {name: 'gcc13',             runson: 'ubuntu-latest',    container: 'ubuntu:noble'}
+          - cfg: {name: 'gcc14',             runson: 'ubuntu-latest',    container: 'ubuntu:noble'}
```

Comment:
This doesn't give gcc14. Probably a good choice for a long term supported distro that will be using gcc14 by default would be `debian:trixie` -- it isn't released yet but is in a hard freeze so they probably will stick with 14